### DISCUSSION TITLE
Upon submission of R code for execution, the code suggestion feature of the positron software fails to operate.

### DISCUSSION THREAD

**Discussion by BRIGHT-YHZ** [posit-dev/positron#5478] [Q&A] [answered]
Hello everyone, I am a newcomer to the positron software platform. I have a question regarding its functionality. Previously, I learned that positron operates similarly to VSCode, in that its operation is not based on an active R session. In other words, like VSCode, we should be able to continue editing code while R is executing, with the code suggestion feature functioning normally. However, during my recent utilization of positron, I have observed that upon code submission for execution, the code suggestion feature of the editor ceases to function, remaining inoperative until the completion of R's execution.

**Comment by petetronic** [comment, marked as answer]
Hi @BRIGHT-YHZ, thank you for trying out a prerelease of Positron! This should significantly improve after a few key improvements are in place. Please feel free to add thoughts to https://github.com/posit-dev/positron/issues/3180

**Reply by BRIGHT-YHZ** [reply]
Thanks for your efforts. I'm looking forward to seeing further improvements in Positron!!!
